src/app/model/validator.model.ts

```diff
@@ -0,0 +1,16 @@
+import { IOperator } from '~app/common/stores/applications/SsvWeb';
```

Comment:
extract IOperator from store to operator.model.ts

Reply:
done